### ISSUE TITLE
Bug in plot decomposition.py

### ISSUE THREAD

**Issue by dfbeltran** [facebookresearch/Kats#243]
When you try to plot (https://github.com/facebookresearch/Kats/blob/main/kats/utils/decomposition.py) it's overlapping the on the ax[0] the original data with the trend, this leaves an empty subplot. This is caused due to in the zip is not considered that it should start from ax[1] onwards

This logic should be reconsidered:
```
for part, ax, title in zip(parts, axs, titles):
    ts: TimeSeriesData = results[part]
    ax.plot(ts.time.values, ts.value.values, linewidth=linewidth)
    ax.set_title(title)
```

**Comment by MoKazemi9** [comment]
@dfbeltran , thanks for the opening this issue, please feel free to submit a PR and the team will surely review it.

**Comment by michaelbrundage** [comment]
Thanks for the bug report. Fix pending.

**Comment by michaelbrundage** [comment]
Fixed in 48b6afb62f9ba334598b9e15f8b87b1a7b0daa4c
